### PR TITLE
Update ConfigFrontEnd0002Test.php

### PR DESCRIPTION
**Pull request by kshitijSharma2014**

### CODE REVIEW

**Comment by kshitijSharma2014** [comment]
@puneet0191  @javigomez 
please review it.
thanks

File: tests/system/webdriver/tests/services/ConfigFrontEnd0002Test.php [hunk 1]
```diff
@@ -78,6 +78,6 @@ public function testChangeMetaDescription()
 	{
 		$this->siteHomePage->changeMetaDescription('JoomlaTestMetaDescription');
 
-		$this->assertNotEquals('JoomlaTestMetaDescription', $this->siteHomePage->getMetaDescription(), 'Site Meta Description has not changed');
+		$this->assertEquals('JoomlaTestMetaDescription', $this->siteHomePage->getMetaDescription(), 'Site Meta Description has not changed');
```
Comment (puneet0191) [inline tests/system/webdriver/tests/services/ConfigFrontEnd0002Test.php]: Please update the Text, It should be 'Meta Description has Changed'

**Comment by puneet0191** [comment]
@kshitijSharma2014 I tested your PR, and I am now getting this result:

Configuration read from /home/puneet/Projects/GSOC-Webdriver_system_tests_for_CMS/tests/system/webdriver/tests/phpunit.xml.dist

FF..

Time: 50.16 seconds, Memory: 3.75Mb

There were 2 failures:

1) ConfigFrontEnd0001Test::testChangeSiteName
Site name has changed
Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
-'Joomla Testing'
+'Joomla'

/home/puneet/Projects/GSOC-Webdriver_system_tests_for_CMS/tests/system/webdriver/tests/services/ConfigFrontEnd0001Test.php:72

2) ConfigFrontEnd0002Test::testChangeMetaDescription
Site Meta Description has changed
Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
-'JoomlaTestMetaDescription'
+''

**Comment by puneet0191** [comment]
@kshitijSharma2014 Closing this one, the changes were not required it is working as expected

**Pull request closed without merging**